src/pat/tooltip/tooltip.js

```diff
@@ -26,7 +26,7 @@ define([
     parser.addArgument("height", "auto", ["auto", "max"]);
     parser.addArgument("trigger", "click", ["click", "hover"]);
     parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
-    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
+    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
```

Comment:
I agree with Rok, we don't know enough about how this is used at the moment so we may break things. We can separate the current task at hand from the work necessary to find out how to unify both into two separate tasks without hurting anybody by adding that second option. We might even decide not to document it for now.